.drone.star

```diff
@@ -1637,6 +1792,27 @@ def setupNotificationsAppForServer():
 	}]
 
 
+def setupServerAndAppsForIntegrationApp(logLevel):
+	return [{
+		'name': 'setup-server-%s' % config['app'],
+		'image': 'owncloudci/php:7.4',
+		'pull': 'always',
+		'commands': [
```

Comment:
Please keep the commands as close to https://owncloud.dev/clients/web/deployments/oc10-app/ as possible - according to those docs, `trusted_domains`, `dav.enable.tech_preview` and `enable_previews` should not be needed, right? Could you try setting it up without those three?

Reply:
(ignore the comment about `enable_previews`, that's of course a good idea for the tests - for the other two I'm actually curious if tests succeed without setting them)

Reply:
#5032 demonstrates that `dav.enable.tech_preview` can be removed.
#5033 removes that from existing CI (that is in master now)

Reply:
thank you @phil-davis for the demonstrations